tests/karma.conf.ts

```diff
@@ -0,0 +1,44 @@
+import { BASE_CONFIG } from '../webpack.config';
+import path from 'path';
+
+module.exports = (config: any) => {
+    config.set({
+        autoWatch: false,
+        browserNoActivityTimeout: 10000,
+        concurrency: 1,
+        client: {
+            mocha: {
+                timeout: 8000,
+            },
+        },
+        exclude: [],
+        files: ['./**/*.test.ts'],
+        frameworks: ['webpack', 'mocha', 'detectBrowsers'],
+
+        preprocessors: {
+            './**/*.test.ts': ['webpack'],
+        },
+        reporters: ['progress'],
+        singleRun: true,
+
+        webpack: BASE_CONFIG,
+
+        webpackMiddleware: {
+            stats: 'errors-only',
+        },
+
+        detectBrowsers: {
+            postDetection: function (availableBrowsers: any) {
+                //Remove IE
+                var result = availableBrowsers;
+                let IEindex = result.indexOf('IE');
+                if (IEindex !== -1) {
+                    result.splice(IEindex);
+                }
+                return result;
+            },
+            preferHeadless: false,
```

Comment:
I'd really like to enable headless testing, I'll check if I can find a way to do it locally

Reply:
Ok.